tests/e2e/specs/stories-editor/page.test.js

```diff
@@ -0,0 +1,183 @@
+/**
+ * WordPress dependencies
+ */
+import { createNewPost, saveDraft, clickButton, selectBlockByClientId, getAllBlocks } from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import {
+	activateExperience,
+	deactivateExperience,
+	clickButtonByLabel,
+	getSidebarPanelToggleByTitle,
+	uploadMedia,
+	openPreviewPage,
+	insertBlock,
+} from '../../utils';
+
+describe( 'Story Page', () => {
+	beforeAll( async () => {
+		await activateExperience( 'stories' );
+	} );
+
+	afterAll( async () => {
+		await deactivateExperience( 'stories' );
+	} );
+
+	beforeEach( async () => {
+		await createNewPost( { postType: 'amp_story' } );
+		// Select the default page block.
+		await selectBlockByClientId(
+			( await getAllBlocks() )[ 0 ].clientId
+		);
+	} );
+
+	it( 'should be possible to add background color to Page', async () => {
+		const panelTitle = 'Background Color';
+		const colorToggle = await getSidebarPanelToggleByTitle( panelTitle );
```

Comment:
Note: the upstream `findSidebarPanelToggleButtonWithTitle` does not seem to work here.

Reply:
[`findSidebarPanelToggleButtonWithTitle`](https://github.com/WordPress/gutenberg/blob/a779901c4fba1d8cd32475d8489708c2a134bb9f/packages/e2e-test-utils/src/find-sidebar-panel-toggle-button-with-title.js) is quite a long name 😄 

Looks like the upstream function doesn't work because the `Background Color` label is wrapped in an additional `.editor-panel-color-settings__panel-title` `<span>`, which it doesn't catch.

Sounds like something that could be reported upstream... Added a note here: https://github.com/WordPress/gutenberg/issues/16675